readme.md

```diff
@@ -59,6 +59,17 @@ services:
       ES_JAVA_OPTS: "-Xms2g -Xmx2g"
 ```
 
+## Xdebug
+
+Add the following lines to `docker-php-ext-xdebug.ini` in order to use Xdebug.
+
+```
+xdebug.remote_enable = 1
+xdebug.remote_host = {YOU_LOCAL_IP_ADDRESS}
```

Comment:
Yes, it has to be you local address in the network and you have to keep it up to date if it changes. If you use 127.0.0.1, your IDE won't be able to use Xdebug, because it runs on a remote machine (outside of the container).